mu-plugins/10up-plugin/plugin.php

```diff
@@ -1,4 +1,20 @@
 <?php
+/**
+ * Plugin Name:       10up Plugin Scaffold
+ * Description:       A brief description of the plugin.
+ * Version:           0.1.0
+ * Requires at least: 4.9
+ * Requires PHP:      7.2
+ * Author:            10up
+ * Author URI:        https://10up.com
+ * License:           GPL v2 or later
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       tenup-plugin
+ * Domain Path:       /languages
+ * Update URI:        https://github.com/10up/wp-scaffold
+ *
```

Comment:
Isn't all this metadata that never actually shows anywhere since mu-plugins are using the plugin loader approach?

Reply:
Correct, though it's not uncommon for someone to copy-paste the mu-plugin as a scaffold for a standard plugin, in which case it's helpful to have this in place.